spec/factories/policies/early_career_payments/eligibilities.rb

```diff
@@ -158,10 +146,10 @@
       nqt_in_academic_year_after_itt { false }
     end
 
-    trait :ineligible do
-      eligible_now
-      ineligible_school
-    end
+    # trait :ineligible do
+    #   eligible_now
+    #   ineligible_school
+    # end
```

Comment:
This commented out trait gets removed in a future pr where we strip out most of the methods in this factory